sdk/management/gulpfile.js

```diff
@@ -150,65 +147,6 @@ function codegen(project, cb) {
     return execa(cmd, [], { shell: true, stdio: "inherit" });
 };
 
-function findTag(stringContainsTag) {
```

Comment:
what're below codes used for?

Reply:
line 112 to 114.

used when autorest v3 does not support `--tag`